fixbackend/cloud_accounts/models/__init__.py

```diff
@@ -38,3 +40,16 @@ class CloudAccount:
     id: CloudAccountId
     workspace_id: WorkspaceId
     access: CloudAccess
+
+
+@frozen
+class LastScanAccountInfo:
+    aws_account_id: str
```

Comment:
The value that is reported is an account id - not necessarily an AWS account id.